src/main/java/ar/com/synergian/wagongit/GitWagon.java

```diff
@@ -27,6 +27,7 @@
 	private final boolean debug = Utils.getBooleanEnvironmentProperty("wagon.git.debug");
 	private final boolean safeCheckout = Utils.getBooleanEnvironmentProperty("wagon.git.safe.checkout");
 	private final boolean skipEmptyCommit = Utils.getBooleanEnvironmentProperty("wagon.git.skip.empty.commit");
+	private final boolean enableShallowFetch = Utils.getBooleanEnvironmentProperty("wagon.git.enable.shallow.fetch");
```

Comment:
Can we invert logic here? What about disableShallowFetch thus we would assume it's enabled by default?